src/components/primitives/Radio/Radio.tsx

```diff
@@ -41,6 +41,13 @@ const Radio = ({ icon, children, ...props }: IRadioProps, ref: any) => {
         justifyContent="center"
         alignItems="center"
         {...newProps}
+        opacity={isDisabled ? 0.4 : 1}
+        {...(Platform.OS === 'web'
+          ? {
+              disabled: isDisabled,
```

Comment:
Would this only be for web?

Reply:
It's only for accessibility reason to let the browser know that the div is disabled, as the cursor will be changing.